f8a_worker/dispatcher/nodes.yml

```diff
@@ -388,6 +395,15 @@
         encryption: false
         versioned: true
 
+    - name: 'S3UserProfileStore'
+      import: 'f8a_worker.storages'
+      configuration:
+        <<: *configurationS3
+        bucket_name: '{DEPLOYMENT_PREFIX}-janus-json-data'
+        region_name: 'us-east-1'
+        encryption: false
+        versioned: true
```

Comment:
I think we would like to encrypt user data (`encryption: true`), not I'm sure about versioning here though.